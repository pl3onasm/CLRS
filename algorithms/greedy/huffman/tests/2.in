aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
bbbbbbbbbbbbb
cccccccccccc
dddddddddddddddd
eeeeeeeee
fffff