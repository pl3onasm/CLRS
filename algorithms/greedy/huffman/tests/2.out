CODEWORD TABLE

char      freq       code
   a        45       0
   c        12       100
   b        13       101
   f         5       1100
   e         9       1101
   d        16       111


HUFFMAN TREE

-0 [45] 'a'
root [100]
-1-0-0 [12] 'c'
-1-0 [25]
-1-0-1 [13] 'b'
-1 [55]
-1-1-0-0 [5] 'f'
-1-1-0 [14]
-1-1-0-1 [9] 'e'
-1-1 [30]
-1-1-1 [16] 'd'

